CONTRIBUTING.md

```diff
@@ -0,0 +1,259 @@
+
+### Creating an Issue
+
+If you experience a problem while using the Raiden WebUI or want to request a feature
+then you should open an issue against the repository. All issues should
+contain:
+
+**For Feature Requests:**
+
+- A description of what you would like to see implemented.
+- An explanation of why you believe this would make a good addition to Raiden WebUI.
+
+**For Bugs:**
+
+- A short description of the problem.
+- Detailed description of your system and environment (e.g. Raiden version, ethereum client etc).
+- What was the exact unexpected thing that occurred.
+- What you were expecting to happen instead.
+
+### Creating a Pull Request
+
+If you have some coding abilities and would like to contribute to the actual
+codebase of Raiden then you can open a Pull Request(PR) against the repository.
+
+If you are interested in contributing make sure that there is an issue about it. Express interest, by picking the issue so that core developers know that you are working on the issue.
+
+All PRs should be:
+
+- Self-contained.
+- As short as possible and address a single issue or even a part of an issue.
+  Consider breaking long PRs into smaller ones.
+
+In order for a Pull Request to get merged into the main repository you should
+have one approved review from one of the core developers of Raiden and also all
+Continuous Integration tests should be passing and the CI build should be
+green.
+
+Additionally you need to sign the raiden project CLA (Contributor License
+Agreement). Our CLA bot will help you with that after you created a pull
+request. If you or your employer do not hold the whole copyright of the
+authorship submitted we can not accept your contribution.
+
+For frequent contributors with write access to the repository we have a set of labels to put on Pull Requests to signal to our colleagues what the current state of the PR is. These are:
+
+- [Dev: Please Review](https://github.com/raiden-network/webui/labels/dev%3A%20Please%20Review) to a Pull Request that is currently ready for a reviewer to have a look at.
+- [Dev: Work in Progress](https://github.com/raiden-network/webui/labels/dev%3A%20Work%20In%20Progress) to a Pull Request that is either not yet ready for review or is getting PR review suggestions applied by the author until it's ready for review again.
+
+### Pull Request Reviews
+
+It is the responsibility of the author to ask for at least one person to review their Pull Request. That person should know the area of the code being changed. If the chosen reviewer does not feel confident in the review, they can then ask for someone else to additionally look at the code.
+
+All the developers in the team should perform Pull Request reviews. Make it a habit to check [this](https://github.com/raiden-network/webui/pulls?q=is%3Apr+is%3Aopen+label%3A%22dev%3A+Please+Review%22) link often to help your fellow colleagues who have PRs open pending for review.
```

Comment:
I would remove this. It should be within the responsibility of the PR author to pull somebody in. External reviewers should just pick a proposed one.